packages/stark-core/src/modules/http/builder/http-abstract-base-request-builder.ts

```diff
@@ -1,7 +1,10 @@
+/* tslint:disable completed-docs*/
```

Comment:
The correct format is: `/* tslint:disable:completed-docs */` otherwise all rules are disabled ;)